sway/input/text_input.c

```diff
@@ -133,6 +141,11 @@ static void relay_send_im_state(struct sway_input_method_relay *relay,
 			input->current.content_type.hint,
 			input->current.content_type.purpose);
 	}
+	struct sway_input_popup *popup;
+	wl_list_for_each(popup, &relay->input_popups, link) {
+		// send_text_input_rectangle is called in this function
+		input_popup_update(popup);
```

Comment:
`input_popup_update()` is a pretty big hammer, it means we'll create/destroy scene nodes every time a keypress is made. But let's keep this as-is for now and leave the optimization for later.